tests/unittests/Foundation/NSSetTests.mm

```diff
@@ -214,16 +213,10 @@
                                       --waitingCount;
                                       [condition signal];
                                       [condition unlock];
-                                      if ([obj unsignedIntegerValue] % 2 == 0) {
-                                          ++timesHit;
-                                          return YES;
-                                      }
-
-                                      return NO;
+                                      return [obj unsignedIntegerValue] % 2 == 0 ? YES : NO;
```

Comment:
The compiler was complaining about not being able to convert bool^(id,BOOL*) to BOOL^(id,BOOL*) so I don't think it's superfluous.